packages/block-editor/src/hooks/style.js

```diff
@@ -31,6 +43,16 @@ const styleSupportKeys = [
 	SPACING_SUPPORT_KEY,
 ];
 
+export const ELEMENTS = {
```

Comment:
Can we instead consolidate the existing `ELEMENTS` in `utils.js` and this one in a common place both the hook and the global styles rendered can access? It looks like `packages/blocks/src/api/constants.js` could be a good option, given we already define there the `__EXPERIMENTAL_STYLE_PROPERTY`.